webui/react/src/pages/ExperimentList.tsx

```diff
@@ -136,15 +124,10 @@ const ExperimentList: React.FC = () => {
   }, [ selectedExperiments ]);
 
   const fetchExperiments = useCallback(async (): Promise<void> => {
-    try {
-      const response = await getExperimentList(sorter, pagination, search, filters);
-      const experiments = decodeExperimentList(response.experiments || []);
-
-      setTotal(response.pagination?.total || 0);
-      setExperiments(experiments);
-    } catch (e) {
-      console.error(e);
-    }
+    const response = await getExperimentList(sorter, pagination, search, filters);
+    const experiments = decodeExperimentList(response.experiments || []);
+    setTotal(response.pagination?.total || 0);
```

Comment:
since the api is going to have pagination (both the request and the response) be the same format we might want to  abstract and reuse that for other places (maybe with the current useRestApi)

actually the current userRestApi hook allows you to update the request parameters whenever needed could that still let us keep using it?